akka-stream/src/main/scala/akka/stream/scaladsl/Materialization.scala

```diff
@@ -15,8 +17,25 @@ object Keep {
   private val _both = (l: Any, r: Any) ⇒ (l, r)
   private val _none = (l: Any, r: Any) ⇒ NotUsed
 
+  private val _leftAsync = (l: Any, r: Any) ⇒ Future.successful(l)
```

Comment:
I'd also vote on not adding those.
You can add them in your tests if you use them somewhere